main-app/app/scripts/game/controllers/game-controller.js

```diff
@@ -0,0 +1,63 @@
+(function () {
+    'use strict';
+
+    angular.module('Tombola.Games.Bingo90.Game').
+        controller('game',
+        ['$scope', '$state', '$interval', 'GameServerProxy', 'RequestShaper', 'UserDetails', 'Ticket', 'CallHandler', 'GameState',
+            function ($scope, $state, $interval, proxy, req, userValue, ticket, callHandler, gameState) {
+                var me = this,
+                    timeBetweenCalls = 100;
+
+                $scope.currentCall = "";
+                $scope.ticket = ticket;
+                $scope.toGo = 5;
+                $scope.callNumber = 0;
+                $scope.currentStage = gameState.currentStage;
+
+                me.nextCall = function () {
+                    $scope.callNumber += 1;
+                    return $scope.callNumber;
+                };
+
+                me.markTicket = function (lastCall) {
+                    ticket.checkMark(lastCall);
+                };
+
+                me.getCall = function () {
+                    proxy.apiCall(req.makeCallRequest(userValue.data.token, me.nextCall())).then(
```

Comment:
Inject whatever is holding the token into the poxy and add it there. 
Not part of the calling code's responsibility to arrange the innards of how the API calls work - this is the job of the proxy.